kubeval linter not used

**Bug description**

When running `make lint`, the tool `kubeval` is being installed but it does not seem to ever be used.
It is a bit inefficient but mostly, it prevents `make lint` from running on `darwin-arm64` because `kubeval` is not available for that architecture; therefore, `make lint` fails when trying to install `kubeval` even though the tool is not needed.

**Affected product area (please put an X in all that apply)**

- ( ) APIs
- ( ) Addons
- ( ) CLI
- ( ) Docs
- ( ) IAM
- ( ) Installation
- ( ) Plugin
- ( ) Security
- (x) Test and Release
- ( ) User Experience
- ( ) Developer Experience

**Expected behavior**

Ultimately, the goal is to have `make lint` pass on `darwin-arm64`,

**Steps to reproduce the bug**

On a Mac M1, run `make lint` and you should see:
```
make -C /Users/kmarc/git/tanzu-framework/hack/tools kubeval
mkdir -p bin
curl -sL https://github.com/instrumenta/kubeval/releases/download/0.15.0/kubeval-darwin-arm64.tar.gz | tar -xz -C /tmp/
tar: Error opening archive: Unrecognized archive format
make[1]: *** [bin/kubeval] Error 1
make: *** [/Users/kmarc/git/tanzu-framework/hack/tools/bin/kubeval] Error 2
```

**Version** (include the SHA if the version is not obvious)

`main` branch: sha 46b4f80a

It seems that the use of `kukbeval` was removed in #204, but the installation of it was not removed.

**Environment where the bug was observed (cloud, OS, etc)**

Locally on Mac M1

--- a/common.mk
+++ b/common.mk
@@ -82,12 +82,11 @@ KBLD               := $(TOOLS_BIN_DIR)/kbld
 VENDIR             := $(TOOLS_BIN_DIR)/vendir
 IMGPKG             := $(TOOLS_BIN_DIR)/imgpkg
 KAPP               := $(TOOLS_BIN_DIR)/kapp
-KUBEVAL            := $(TOOLS_BIN_DIR)/kubeval
 GINKGO             := $(TOOLS_BIN_DIR)/ginkgo
 VALE               := $(TOOLS_BIN_DIR)/vale
 YQ                 := $(TOOLS_BIN_DIR)/yq
 CONVERSION_GEN     := $(TOOLS_BIN_DIR)/conversion-gen
-TOOLING_BINARIES   := $(CONTROLLER_GEN) $(GOLANGCI_LINT) $(YTT) $(KBLD) $(VENDIR) $(IMGPKG) $(KAPP) $(KUBEVAL) $(KUSTOMIZE) $(GOIMPORTS) $(GOBINDATA) $(GINKGO) $(VALE) $(YQ) $(CONVERSION_GEN)
+TOOLING_BINARIES   := $(CONTROLLER_GEN) $(GOLANGCI_LINT) $(YTT) $(KBLD) $(VENDIR) $(IMGPKG) $(KAPP) $(KUSTOMIZE) $(GOIMPORTS) $(GOBINDATA) $(GINKGO) $(VALE) $(YQ) $(CONVERSION_GEN)
 
 ## --------------------------------------
 ##@ API/controller building and generation

--- a/hack/tools/Makefile
+++ b/hack/tools/Makefile
@@ -29,7 +29,6 @@ YTT_VERSION=v0.40.0
 KBLD_VERSION=v0.31.0
 KAPP_VERSION=v0.42.0
 VENDIR_VERSION=v0.23.0
-KUBEVAL_VERSION=0.15.0
 VALE_VERSION=2.17.0
 K8S_VERSION=1.23.5
 
@@ -43,7 +42,6 @@ GINKGO             := $(BIN_DIR)/ginkgo
 GOLANGCI_LINT      := $(BIN_DIR)/golangci-lint
 GOIMPORTS          := $(BIN_DIR)/goimports
 KUBEBUILDER        := $(BIN_DIR)/kubebuilder
-KUBEVAL            := $(BIN_DIR)/kubeval
 KUSTOMIZE := $(BIN_DIR)/kustomize
 GOBINDATA          := $(BIN_DIR)/gobindata
 VALE               := $(BIN_DIR)/vale
@@ -97,13 +95,6 @@ $(KUBEBUILDER):
 kustomize: $(KUSTOMIZE)
 $(KUSTOMIZE): $(BIN_DIR) go.mod go.sum # Build kustomize from tools folder.
 	CGO_ENABLED=0 go build -tags=tools -o $@ sigs.k8s.io/kustomize/kustomize/v4
-
-kubeval: $(KUBEVAL) ## Install kubeval
-$(KUBEVAL):
-	mkdir -p $(BIN_DIR)
-	curl -sL https://github.com/instrumenta/kubeval/releases/download/$(KUBEVAL_VERSION)/kubeval-$(HOST_OS)-$(HOST_ARCH).tar.gz | tar -xz -C /tmp/
-	mv /tmp/kubeval $(@)
-	chmod a+x $(@)
 
 gobindata: $(GOBINDATA)
 $(GOBINDATA): go.mod go.sum # Build go-bindata

